### DISCUSSION TITLE
Version 10 Foundry Compatibility issues (Delete please)

### DISCUSSION THREAD

**Discussion by HighKingKota** [JohanFalt/Foundry_WoD20#289] [Q&A] [answered]
I can't open the Character sheet when I select actors. Is there something I can do? Thank you.

**Comment by JohanFalt** [comment]
As the system doesn’t support v10 yet you need to downgrade Foundry again to v9.

**Comment by JohanFalt** [comment]
Working on it but it is a few weeks of so I hope you have a v9 backup.

**Comment by JohanFalt** [comment]
The v10 version of the system will be v2 (so you know)

**Reply by HighKingKota** [reply]
Thank you. I didn't see this was already brought up in the issues section. Thanks for the quick response. i don't have a backup of v9 because of hubris.

**Comment by JohanFalt** [comment]
I have so decided to rearrange my work and will start look into this.

**Reply by HighKingKota** [reply]
you rock dude!

**Comment by JohanFalt** [comment, marked as answer]
Just released v2.0.0 of my World of Darkness 20th System which is the version for Foundry v10, it will NOT work on Foundry v9.

This is a beta release which I need help to test. All from what I can see things should work but I need milage on this to be sure. Depracated that comes up as warning are not interesting in this stage, just pure errors and things that doesn't work.
For details for this version see https://github.com/JohanFalt/Foundry_WoD20/wiki/Changelog#fix-in-200
Any problems or comments leave these at https://github.com/JohanFalt/Foundry_WoD20/issues as usual

And ALWAYS BACKUP YOUR FOUNDRY before updating!

Happy Gaming!!
